Refine Microk8s and Multipass docs

RFMVasconcelos opened the pull request:
The intro on these docs was confusing and outdated regarding new Microk8s capabilities.

RFMVasconcelos commented:
/assign @jlewi

joeliedtke commented:
/assign

joeliedtke commented:
/ok-to-test

joeliedtke commented:
Preview: https://deploy-preview-2019--competent-brattain-de2d6d.netlify.app/docs/started/workstation/getting-started-multipass/

joeliedtke commented on content/en/docs/started/workstation/getting-started-multipass.md:
```diff
@@ -7,21 +7,15 @@ weight = 60
 ## Introduction
 
-If you already have Ubuntu or Linux that supports [snaps](https://snapcraft.io/), you can easily install Kubernetes using [Microk8s](https://microk8s.io/). You can jump to the **Install Kubeflow using Microk8s** section below.
+This document outlines the steps that you can take to get your local installation of Kubeflow running on top of [Microk8s](https://microk8s.io/), a small enterprise Kubernetes cluster. Microk8s is now available on Windows, MacOs and any Linux distribution that supports `snaps`. You can download it on the [Microk8s](https://microk8s.io/) website. 
```
MacOs and -> macOS, and

RFMVasconcelos replied:
Nice catch!

joeliedtke commented on content/en/docs/started/workstation/getting-started-multipass.md:
```diff
@@ -7,21 +7,15 @@ weight = 60
+This document outlines the steps that you can take to get your local installation of Kubeflow running on top of [Microk8s](https://microk8s.io/), a small enterprise Kubernetes cluster. Microk8s is now available on Windows, MacOs and any Linux distribution that supports `snaps`. You can download it on the [Microk8s](https://microk8s.io/) website. 
 
-If you don't have a Linux system already, or you would like to confine your Kubeflow to a disposable machine, then [Create a VM with Multipass](https://multipass.run/) first and then follow the instructions below. That will get you an Ubuntu machine that can be used to install Kubernetes and Kubeflow.
+Alternatively, you can install Microk8s on a Linux appliance using [Multipass](https://multipass.run/), if you would like to confine your Kubeflow to a disposable Linux virtual machine on your Windows or MacOS host.
```
Suggested revision:

Alternatively, to deploy Kubeflow within a displosable Linux virtual machine, you can install Microk8s on a Linux appliance using [Multipass](https://multipass.run/) on Windows or MacOS.

RFMVasconcelos commented:
/assign @joeliedtke

8bitmp3 commented on content/en/docs/started/workstation/getting-started-multipass.md:
```diff
@@ -7,21 +7,15 @@ weight = 60
 ## Introduction
 
-If you already have Ubuntu or Linux that supports [snaps](https://snapcraft.io/), you can easily install Kubernetes using [Microk8s](https://microk8s.io/). You can jump to the **Install Kubeflow using Microk8s** section below.
+This document outlines the steps that you can take to get your local installation of Kubeflow running on top of [Microk8s](https://microk8s.io/), a small enterprise Kubernetes cluster. Microk8s is now available on Windows, MacOs and any Linux distribution that supports `snaps`. You can download it on the [Microk8s](https://microk8s.io/) website. 
```
I propose to simplify your first sentence by cutting some words out, while retaining the meaning. At the moment, it's a bit verbose: "This doc outlines the steps that you can take to..."

```diff
This 
- document outlines the steps that you can take 
+ guide will show you how
- to get your local 
+ to deploy Kubeflow
- installation of Kubeflow 
- running on top of 
+ on [Microk8s](https://microk8s.io/)
- , a small enterprise Kubernetes cluster.
+ —a small enterprise Kubernetes cluster.
```

RFMVasconcelos replied:
Awesome! Thank you for this review 🚀

RFMVasconcelos commented:
@8bitmp3 could I ask you for a final review on this? I have implemented your suggestions in the last commit. 

I will be adding a few more PRs today to get these microk8s docs on point 🚀

8bitmp3 commented on content/en/docs/started/workstation/getting-started-multipass.md:
```diff
@@ -10,7 +10,7 @@ weight = 60
 ## Introduction
 
-This document outlines the steps that you can take to get your local installation of Kubeflow running on top of [Microk8s](https://microk8s.io/), a small enterprise Kubernetes cluster. Microk8s is now available on Windows, macOS and any Linux distribution that supports `snaps`. You can download it on the [Microk8s](https://microk8s.io/) website. 
+This guide will show you how to deploy Kubeflow using [Microk8s](https://microk8s.io/) - a small enterprise Kubernetes cluster. Microk8s is now available on Windows, macOS and any Linux distribution that supports `snaps`. You can download it on the [Microk8s](https://microk8s.io/) website. 
```
After some studying, I realized that using a present tense is the commonly used practice.

```diff
- This guide will show you how to to deploy Kubeflow using [Microk8s]
+ This guide describes how to deploy Kubeflow using [Microk8s]
```

What do you think?

RFMVasconcelos replied:
LGTM

RFMVasconcelos replied:
Thank you!

RFMVasconcelos commented:
Ended up doing all the changes on this PR for simplicity. I believe it is ready to merge.

8bitmp3 commented:
@RFMVasconcelos I'll create a PR for your repo and explain the changes in detail here, so we can finalize this and get closer to the merge 👍

RFMVasconcelos commented:
Hi @8bitmp3, any update on this front? 🚀 

Which changed do you think are still needed?

RFMVasconcelos commented:
@joeliedtke, any chance you could review + approve this?

8bitmp3 commented:
@RFMVasconcelos I split the installation and enabling steps in digestible chunks. Introduced other minor changes. LMKWYT! https://github.com/RFMVasconcelos/website/pull/1 (By the way, I noticed it's "MicroK8s" instead of "Microk8s" - when changing the format automatically, it also affected the command lines (`MicroK8s...`) - can you fix them if they affect the output).

RFMVasconcelos commented:
@8bitmp3 Thank you for the help here! 

It seems that you started editing from an outdated commit, not this branch, so you reverted a few changes. Let me cherrypick from your changes the ones that are new and commit them directly here. sounds good?

Cheers!

8bitmp3 commented:
Absolutely @RFMVasconcelos The dated branches really showed 🙂 👍

8bitmp3 commented:
@RFMVasconcelos As discussed a few hours ago, since you don't have enough time today - see PR: https://github.com/RFMVasconcelos/website/pull/2 for https://github.com/kubeflow/website/pull/2019.

I studied the MicroK8s and Multipass docs again to make sure we don’t miss anything here. Some of the significant changes proposed:

- Change: “Microk8s” -> “MicroK8s”
- Add: Multipass gives an Ubuntu command line - and you can install MicroK8s on top of it
    - Add Multipass docs link, since the installation process for Muiltipass + MicroK8s is lengthier than for native MicroK8s
- Add :Multipass also supports Linux
- More sequential steps in Accessing the dashboard section
- Clarify: replace IP placeholders with own IP addresses (e.g. `<multipass_public_ip>`)
- Bolden: the UI elements in Linux
- Change: “Next steps” -> “Additional guides”

```diff
- description = “Quickly get Kubeflow running locally on built-in hypervisors with MicroK8s"
+ description = "Run Kubeflow locally on built-in hypervisors with MicroK8s" weight = 60
…
This guide describes how to deploy Kubeflow using Microk8s - a small enterprise Kubernetes cluster
+ - locally on built-in hypervisors with MicroK8s.
```

8bitmp3 commented:
/lgtm

RFMVasconcelos commented:
@8bitmp3 I made a few final changes after your merge.

RFMVasconcelos commented:
/ok-to-test

RFMVasconcelos commented:
/lgtm

RFMVasconcelos commented:
@8bitmp3 I fixed text wrapping and also removed the alpha status, as this applies to applications, according to [version policy](https://www.kubeflow.org/docs/reference/version-policy/), not deployment methods.

RFMVasconcelos commented:
WDYT?

8bitmp3 commented on content/en/docs/started/workstation/getting-started-multipass.md:
```diff
@@ -42,55 +33,48 @@ it, and then set up Kubeflow.
 
-    **Optional:** To enable NVIDIA GPU hardware support, also run
-    `MicroK8s.enable gpu`.
+    **Optional:** To enable NVIDIA GPU hardware support, also run `MicroK8s.enable gpu`.
```
@RFMVasconcelos Can you please check if it's `microK8s.enable gpu` (lower caps)? if this works too, no worries

RFMVasconcelos replied:
Oh yeah, lower is better!

8bitmp3 commented on content/en/docs/started/workstation/getting-started-multipass.md:
````diff
@@ -42,55 +33,48 @@ it, and then set up Kubeflow.
     ```
 
-    Or, you can check for the Multipass IP first with:
+    On Windows, macOS or multipass, you can check for the IP first with:
````
🙂 @RFMVasconcelos 

<img width="500" alt="image" src="https://user-images.githubusercontent.com/19637339/92277339-9b17af00-eeea-11ea-98b8-d8f48bb9715d.png">

RFMVasconcelos replied:
Hahah fixing!

RFMVasconcelos commented:
Hey @8bitmp3 @joeliedtke!

It seems we finally got to a good point in this doc. 🚀  

Could I gently ask you to review/approve this PR?

8bitmp3 commented on content/en/docs/started/workstation/getting-started-multipass.md:
```diff
@@ -1,97 +1,99 @@
-* Finally, (1) open a new web browser window and (2) access the link provided after you enable Kubeflow, e.g. `10.64.140.43.xip.io` (see previous step).
+4. Finally, access the Kubeflow dashboard by:
+    1. Opening a new web browser window.
+    2. Access the link provided after you have enabled Kubeflow (for example, `10.64.140.43.xip.io`).
```
Maybe this should be "Accessing". Other than that, lgtm

RFMVasconcelos replied:
Nice catch! Solved! 🚀

RFMVasconcelos commented:
@joeliedtke @8bitmp3 I am not able to approve this, as it's my PR, could you please approve this?

8bitmp3 commented:
/ok-to-test

8bitmp3 commented:
/lgtm
/approve
